demo/App.js

```diff
@@ -1,12 +1,177 @@
 import React from "react";
 import { StyleSheet, Text, View } from "react-native";
 import { AppLoading, Asset, Font } from "expo";
+import "ethers/dist/shims.js";
+import { ethers } from "ethers";
```

Comment:
```suggestion
import { ethers } from "ethers";

// Note: In the future, we'll be importing the Tasit SDK
// which has ethers.js as a dependency rather than importing
// it directly
```